### DISCUSSION TITLE
🤗 Customize Resource Feature

### DISCUSSION THREAD

**Discussion by nyomansunima** [nitrictech/nitric#491] [Ideas]
Hey, thank to make this awesome tools, I really like this. But i have some idea to make this project can be used by everyone else from the different background and situations.

I was a indie hackers, and want to build something amazing, including trial and error products. Let's say i don't have any money to start. that's why i should use another service than aws, gcp or azure to get the free version of service. 

Will be better if there option to manage the project service to use by the user. for example the bucket, serverless, and other.

Thanks. Let's make this better everyday.

**Comment by jyecusch** [comment]
Hey @nyomansunima thanks for the feedback and I'm glad you're liking nitric. I think what you're asking for is the ability to deploy to providers other than AWS, GCP and Azure. We've been working toward that goal recently with the addition of [custom provider](https://nitric.io/docs/reference/providers/custom/building-custom-provider) support.

The nitric providers are all plugin based, so it's possible to build your own now. We'd be happy to provide assistance too if you need it. If you're looking for something different feel free to create an issue for a feature request or start a PR for your own feature.
